lighthouserc.json

```
@@ -0,0 +1,21 @@
+{
+    "ci": {
+      "collect": {
+        "startServerCommand": "npm start",
+        "url": ["http://localhost:4000/", "http://localhost:4000/topics/", "http://localhost:4000/resources/", "http://localhost:4000/law-and-regs/", "http://localhost:4000/cases/", "http://localhost:4000/file-a-complaint/"],
```

nit - can you move these to one entry per line?

It makes it a bit more readable, and helps avoid merge conflicts when we add/changes pages

```suggestion
        "url": [
            "http://localhost:4000/", 
            "http://localhost:4000/topics/",
            "http://localhost:4000/resources/",
            "http://localhost:4000/law-and-regs/",
            "http://localhost:4000/cases/",
            "http://localhost:4000/file-a-complaint/"
        ],
```